scripts/validate/govcms-validate-permissions

```diff
@@ -68,7 +68,7 @@ for file in $GOVCMS_FILE_LIST; do
     FAILURES="$FAILURES,$file"
     continue
   fi
-  if [[ $(yq r "$file" 'is_admin') == 'true' ]]; then
+  if [[ $(yq r "$file" 'is_admin') != 'false' ]]; then
```

Comment:
We had issues with this for some checks - Drupal looks like it can output truthy's rather than explicit `true` or `false` values.

https://github.com/salsadigitalauorg/shipshape/commit/b92caba4476940293a045e59a6488b9a83bb3379

Reply:
This swaps explicit check for `true`, which allowed `1` to slip through, to an explicit check for `false` to avoid that. So perhaps not creating any new problems?

That being said, drupal exports `is_admin` set to `null` from what I know, which means that this check will not pass. Should we explicitly add `&& != 'null'`?